packages/create/index.ts

```diff
@@ -98,7 +98,7 @@ function matches_condition(condition: Condition, options: Options) {
 	if (condition === 'demo' || condition === 'minimal' || condition === 'library') {
 		return options.template === condition;
 	}
-	if (condition === 'typescript' || condition === 'checkjs') {
+	if (condition === 'typescript' || condition === 'checkjs' || condition === 'none') {
 		return options.types === condition;
 	}
 	return !!options[condition];
```

Comment:
would this be equivalent? I think all possible values are in this `if` statement
```suggestion
	return options.types === condition;
```

Reply:
not exactly since it's seemingly possible for the condition to exist as a key of `options`. i may be able to simplify it a tiny bit though